test_cases/solidity/insufficient_gas_griefing/relayer/relayer.sol

```diff
@@ -0,0 +1,27 @@
+/*
+ * @source: https://consensys.github.io/smart-contract-best-practices/known_attacks/#insufficient-gas-griefing
+ * @author: ConsenSys Diligence
+ * Modified by Kaden Zipfel
+ */
+
+pragma solidity ^0.4.21;
+
+contract Relayer {
+    mapping (bytes => bool) executed;
+
+    function relay(bytes _data) public {
+        // replay protection; do not call the same transaction twice
+        require(executed[_data] == false);
+        executed[_data] = true;
+
+        address executor = new Executor();
+        executor.call(bytes4(keccak256("execute(bytes)")), _data);
+    }
+}
+
+// Contract called by Relayer
+contract Executor {
```

Comment:
This name feels misleading to me. 
Perhaps something like Target could work, but I'm open to something even better.